gravitee-apim-console-webui/src/management/api/notifications/notifications-list/notofication-details/notification-details.module.ts

```diff
@@ -15,12 +15,48 @@
  */
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { GioFormFocusInvalidModule, GioFormHeadersModule, GioIconsModule, GioSaveBarModule } from '@gravitee/ui-particles-angular';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCardModule } from '@angular/material/card';
+import { MatTableModule } from '@angular/material/table';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatGridListModule } from '@angular/material/grid-list';
 
 import { NotificationDetailsComponent } from './notification-details.component';
 
+import { GioPermissionModule } from '../../../../../shared/components/gio-permission/gio-permission.module';
+import { GioGoBackButtonModule } from '../../../../../shared/components/gio-go-back-button/gio-go-back-button.module';
+
 @NgModule({
   declarations: [NotificationDetailsComponent],
   exports: [NotificationDetailsComponent],
-  imports: [CommonModule],
+  imports: [
+    CommonModule,
+    GioPermissionModule,
+    GioIconsModule,
+    GioPermissionModule,
+    GioSaveBarModule,
+    GioFormHeadersModule,
+    GioFormFocusInvalidModule,
+    GioGoBackButtonModule,
+    ReactiveFormsModule,
+    MatButtonModule,
+    MatCardModule,
+    MatTableModule,
+    MatSnackBarModule,
+    MatAutocompleteModule,
```

Comment:
Are you sure that all these modules are necessary? MatAutocompleteModule, MatGridListModule, MatDialogModule, ...?

Reply:
Done